input/fsh/Bundle.fsh

```diff
@@ -21,14 +21,51 @@ Description: "A collection of data for an mCODE cancer patient."
     tumorMarker 0..* MS and
     cancerRelatedMedicationRequest 0..* MS and
     cancerRelatedProcedure 0..* MS and
-    // Multiple profiles can be used to represent cancer stage. We will combine them into a single element for simplicity.
     cancerStage 0..* MS and
-    // mCODE genomics
     cancerGeneticVariant 0..* MS and
     cancerGenomicsReport 0..* MS and
     geneticSpecimen 0..* MS and
-    genomicRegionStudied 0..* MS
-    // TODO handle comorbidities, waiting on decision for how these are profiled
+    genomicRegionStudied 0..* MS and
+    cancerRelatedComorbidities 0..* MS and
+    vitalSign 0..* MS and
+    coreLaboratory 0..* MS and
+    otherResources 0..1
+
+* entry[cancerPatient] ^short = "Cancer Patient"
+* entry[primaryCancerCondition] ^short = "Primary Cancer Condition"
+* entry[secondaryCancerCondition] ^short = "Secondary Cancer Condition(s)"
+* entry[cancerDiseaseStatus] ^short = "Cancer Disease Status(es)"
+* entry[performanceStatus] ^short = "Performance Status(es) (ECOG or Karnofsky)"
+* entry[tumorMarker] ^short = "Tumor Marker Tests"
+* entry[cancerRelatedMedicationRequest] ^short = "Cancer-RelatedMedicationRequest(s)"
+* entry[cancerRelatedProcedure] ^short = "Cancer-Related Procedure(s)"
+* entry[cancerStage] ^short = "Cancer Stage Data"
+* entry[cancerGeneticVariant] ^short = "Cancer Genetic Variant(s)"
+* entry[cancerGenomicsReport] ^short = "Cancer Genomics Report(s)"
+* entry[geneticSpecimen] ^short = "Genetic Specimen(s)"
+* entry[genomicRegionStudied] ^short = "Genomic Region(s) Studied"
+* entry[cancerRelatedComorbidities] ^short = "Cancer-Related Comorbities"
+* entry[vitalSign] ^short = "Vital Sign"
+* entry[coreLaboratory] ^short = "CBC and CMP Laboratory Results"
```

Comment:
Do we define what "core" means? And are we really limiting to CBC and CMP, or should this say "for example, CBC and CMP..."?

Reply:
Core means CBC and CMP with specific LOINC codes. I haven't found a way to loosen this and still have the slicing work.

Reply:
We can stick with this until we find a better approach.